universe/api/api.go

```diff
@@ -33,3 +33,9 @@ func Initialize(ctx context.Context, cfg *config.Config) error {
 
 	return nil
 }
+
+func AbortRequest(c *gin.Context, code int, reason string, err error) {
+	c.AbortWithStatusJSON(code, gin.H{
+		"error": map[string]string{"reason": reason, "message": err.Error()},
```

Comment:
what is it? we have standardized error responses, let's use struct

Reply:
Structs added